examples/with-apollo-auth/README.md

```diff
@@ -47,23 +47,20 @@ This is an extention of the _[with Apollo](https://github.com/zeit/next.js/tree/
 
-[graph.cool](https://www.graph.cool) can be setup with many different
-[authentication providers](https://www.graph.cool/docs/reference/integrations/overview-seimeish6e/#authentication-providers), the most basic of which is [email-password authentication](https://www.graph.cool/docs/reference/simple-api/user-authentication-eixu9osueb/#email-and-password). Once email-password authentication is enabled for your graph.cool project, you are provided with 2 useful mutations: `createUser` and `signinUser`.
+On loading each route, we perform a `me` query to see if the current visitor is logged in (based on a cookie, more on that in a moment). Depending on the query result, and the route, the user may be [redirected](https://github.com/zeit/next.js/blob/master/examples/with-apollo-auth/lib/redirect.js) to a different page.
 
-On loading each route, we perform a `user` query to see if the current visitor is logged in (based on a cookie, more on that in a moment). Depending on the query result, and the route, the user may be [redirected](https://github.com/zeit/next.js/blob/master/examples/with-apollo-auth/lib/redirect.js) to a different page.
+When creating an account `register` mutation are executed in `/api/graphql`, which returns a token that can be used to authenticate the user. The token is stored in a cookie for easy access (_note: This may have security implications. Please understand XSS and JWT before deploying this to production_).
 
-When creating an account, both the `createUser` and `signinUser` mutations are executed on graph.cool, which returns a token that can be used to [authenticate the user for future requests](https://www.graph.cool/docs/reference/auth/authentication-tokens-eip7ahqu5o/). The token is stored in a cookie for easy access (_note: This may have security implications. Please understand XSS and JWT before deploying this to production_).
+A similar process is followed when using `login` mutation.
 
-A similar process is followed when signing in, except `signinUser` is the only mutation executed.
+It is important to note the use of Apollo's `client.cache.reset()` method after signing in and signing out to ensure that no user data is kept in the browser's memory.
 
-It is important to note the use of Apollo's `resetStore()` method after signing in and signing out to ensure that no user data is kept in the browser's memory.
-
-To get this example running locally, you will need to create a graph.cool
-account, and provide [the `project.graphcool` schema](https://github.com/zeit/next.js/blob/master/examples/with-apollo-auth/project.graphcool).
+To get this example running locally, you will need to create a Postgres database, and database url in [the `schema.prisma` file](https://github.com/zeit/next.js/blob/master/examples/with-apollo-auth/prisma/schema.prisma).
+Photon supports other databases if you don't wish to use Postgres see [Photon](https://photonjs.prisma.io/) for more information.
 
```

Comment:
Might be good to add dummy login credentials if we're going to leave this database in the example 🤔